src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = '';
+  }
+
+  render() {
+    return <Calculator />;
+  }
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/199395761-d4e38aa7-a5d8-49ee-82a4-55687d3401a3.png)

**[Optional]**

- [ ] In your readme file there is a **Getting Started** Section you have added some commands as you can see in the image above :arrow_up:. Kindly add `npm install` command after the clone command to make your application's dependencies easily installable for other non-tech or newbies-in-tech people :+1: